jekyll/_cci2/configuration-cookbook.md

```diff
@@ -44,7 +44,7 @@ For more detailed information about CircleCI orbs, please refer to the [Orbs Int
 
 `version: 2.1`
 
-**Note:** If you do not already have pipelines enabled, go to **Project Settings -> Advanced Settings** and enable them.
+**Note:** If you do not already have pipelines enabled, go to **Project Settings**, select settings for the project you are currently working on by clicking on the gear icon, select **Advanced Settings** and scroll down to enable pipelines.
```

Comment:
I expanded this step to be accurate to what users need to do to get to the right place in the UI, but I think these instructions appear repeatedly in the Orbs Ref. I'm sure people work it out but just wanted to point out there's a few more steps involved.